sdk-setup/src/sdk-freespace

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Copyright (c) 2021, Jolla Ltd. 
+# All rights reserved. 
+#  
+# Redistribution and use in source and binary forms, with or without 
+# modification, are permitted provided that the following conditions are met: 
+#     * Redistributions of source code must retain the above copyright 
+#       notice, this list of conditions and the following disclaimer. 
+#     * Redistributions in binary form must reproduce the above copyright 
+#       notice, this list of conditions and the following disclaimer in the 
+#       documentation and/or other materials provided with the distribution. 
+#     * Neither the name of the <organization> nor the 
+#       names of its contributors may be used to endorse or promote products 
+#       derived from this software without specific prior written permission. 
+#  
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND 
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED 
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE 
+# DISCLAIMED. IN NO EVENT SHALL <COPYRIGHT HOLDER> BE LIABLE FOR ANY 
+# DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES 
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND 
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT 
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS 
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+df --output=avail / | tail -n 1 > /host_targets/df.cache
```

Comment:
I would prefer `/etc/mersdk/share/df.cache`.

Reply:
That was my first thought as well, but as /etc/mersdk/share is not writable, I thought host_targets would do. Of we can change /etc/mersdk/share to be writable, if that's what you really want?

Reply:
hm, I wasn't sure about it being RW (and I couldn't check from inside the build engine at that moment), so I checked just the sources https://github.com/sailfishos/sdk-setup/blob/master/sdk-setup/systemd/etc-mersdk-share.service#L22 before I suggested that. Now I see the installer configures the VBox share as RO, so that explains it. Yup, I think it make sense to let it be RW.